### ISSUE TITLE
Issue With React-Native and `can-promise`

### ISSUE THREAD

**Issue by emyann** [soldair/node-qrcode#157]
Hello, thank you for this handy package.

I was able to use it with react-native in the Android Simulator, but I am actually facing an issue on my S8 Galaxy (Android v8.0.0)

```
undefined is not an object (evaluating 'G.Promise')

Stack trace:
  node_modules/can-promise/index.js:7:13 in exports
  node_modules/qrcode/lib/browser.js:11:23 in renderCanvas
```

It seems to be coming from the dependency `can-promise` https://github.com/vigreco/can-promise/blob/master/index.js#L7 , Also I don't call renderCanvas in my code, just `QRCode.toString()`

I am not sure how to tackle this, some ideas ?

Thanks in advance

**Comment by soldair** [comment]
sorry didn't mean to close this

**Comment by soldair** [comment]
published a new version 1.3.1 that shouldn't throw when testing for Promise.
let me know how it goes and reopen if there is another issue. =)
